libs/wire-subsystems/test/unit/Wire/UserStoreSpec.hs

```diff
@@ -20,7 +20,7 @@ spec = do
 
     prop "user deleted" $ \domain defaultLocale storedUser ->
       let user = mkUserFromStored domain defaultLocale storedUser
-       in user.userDeleted === (storedUser.status == Just Deleted)
+       in userDeleted user === (storedUser.status == Just Deleted)
```

Comment:
that's was `HasField` is for, right?  :)

(please don't, though. i'm growing more and more skeptical about the whole RecordDots thing.)